Add-ons/UmbracoForms/Developer/Configuration/index.md

```diff
@@ -48,11 +53,13 @@ This setting needs to be a `True` or `False` value and will enable the ASP.NET A
 If you do set this to `True` then you need to add `@Html.AntiForgeryToken()` to your forms. The default template for Forms can be found in `~/Views/Partials/Forms/Form.cshtml` and should have `@Html.AntiForgeryToken()` in the `@using (Html.BeginUmbracoForm [...]` block.
 
 ### StoreUmbracoFormsInDb
-
 This setting needs to be set to `True` if you want your Forms data to be stored in the database instead of the .json files in the `App_Data/UmbracoForms` directory in the file system.
 
 For more information on this, read the [Forms in the Database](../Forms-in-the-Database) article.
 
+### UseLegacyPageService 
+In version 8.7.0 an amend was made to improve performance of a service responsible for retrieving the content of the Umbraco page where a form is hosted, in order to allow for the population of string placeholders with the values of properties from the page.  It's expected that the newer implementation will be the best one to use in all scenarios, as, although there are subtle behaviour changes, they aren't expected to be of concern in how the service is used.  However, if there is any reason to use the old one, it can be reinstated by setting this configuration value to `True`.
```

Comment:
I've made some significant changes to this part.
First of all, I add a full stop in the first sentence, as it exceeded our recommendation of a max length of 40 words. Could you let me know if the sentence still makes sense?

Second, I removed most of the final part, as I think the wording sounded a little vague as if we do not really believe that the change will actually work 😅 
Happy to add more details if it's necessary - but I think keeping it short and clean is the best option. What do you think?

```suggestion
In Umbraco Forms 8.7 an update was made to improve the performance of the service responsible for retrieving the content of the Umbraco page where a form is hosted. This was done in order to allow for the population of string placeholders with the values of properties from the page.  

By setting the value of the `UseLegacyPageService` to `True` the old service can be reinstated.
```

Reply:
Happy with most of these changes but I think it's now a little unclear in the second sentence, as it sounds now that the update was made for the reason described, but actually I was trying to explain what the updated service is used for.

So perhaps change the first paragraph to the following, and ths second as you have suggested.:

```
In Umbraco Forms 8.7 an update was made to improve the performance of the service responsible for retrieving the content of the Umbraco page where a form is hosted. This service is used the population of string placeholders - or "magic strings" - with the values of properties from the page.
```

Reply:
Thanks for this @AndyButland !
Seems that some words might have gotten lost in that sentence
`This service is used the population of string placeholders - or "magic strings" - with the values of properties from the page.`

Should it instead be
`This service is using the population of string ...`
or perhaps
`This service is used to populate the string ...` ? 🤔

Reply:
What I meant was `This service is used in the population of....".  Sorry!  But your second suggestion if fine (probably better).

Reply:
Great! Thanks 😄 
I'll update the article 💪